app/src/main/res/layout/logic_analyzer_select_channel.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/channel_select_mode_ll"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <Spinner
+        android:id="@+id/channel_mode_select_spinner"
+        android:layout_width="fill_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="10dp"
+        android:gravity="center_horizontal" />
+
+</LinearLayout>
```

Comment:
Hello @viveksb007 ! This is a nice fix! Good job! :100: 
By adding a new line at the end of file, this warning can be fixed! :+1: